app/views/organization/editemployee.html

```diff
@@ -64,7 +64,15 @@ <h4>{{'label.heading.editemployee' | translate}}</h4>
                         <label class="control-label col-sm-2" for="mobileNo">{{ 'label.input.mobileNo' | translate }}</label>
 
                         <div class="col-sm-3">
-                            <input type="text" id="mobileNo" class="form-control" ng-model="formData.mobileNo">
+                            <input type="text" id="mobileNo" name="mobileNo" ng-model="formData.mobileNo"
```

Comment:
Shouldn't we change the `type` to `number` so that there are minimum chances of the user to enter something else?

Reply:
@luckyman20 my first attempt was that but input type `number` will leave the input box with up and down buttons ![image](https://user-images.githubusercontent.com/29277048/91167271-59158e80-e6dc-11ea-8ead-e71beabf8be0.png) which is of no use